docs/CHANGELOG.rst

```diff
@@ -17,6 +17,12 @@ Added
   process
 * Mechanism to override the manager's control channel prefix from the
   environment
+* Add processes ``abstract-alignment``, ``abstract-annotation``,
```

Comment:
I suggest: Add abstract processes ``abstract-alignment``, ``abstract-annotation``, ``abstract-expression``, ``abstract-differentialexpression`` and ``abstract-bed``. With abstract processes, we are making sure that all subtype processes have the input and output fields that are defined in the abstract process. For users, it is much more clear and predictable if they can expect the same set of outputs on all processes of the same type.